src/components/Input/QtySelector/QtySelector.js

```diff
@@ -90,11 +119,11 @@ class QtySelector extends Component {
   };
 
   renderInput = () => {
-    const { disabled, style, ...rest } = this.props;
+    const { disabled, style, min, max, ...rest } = this.props;
     const { value, focused } = this.state;
     // rendering multiple inputs is required for animation
     return !focused ? (
-      [...Array(100)].map((_, i) => (
+      [...Array(max - min + 1)].map((_, i) => (
```

Comment:
🙌 Seems like a solid improvement. Ideally, it should only either render contiguous numbers or use a single number that has its inner text updated, but that is out of scope for your requirements.